tools/bgpreader.c

```diff
@@ -596,13 +644,26 @@ int main(int argc, char *argv[])
     goto err;
   }
 
+#ifdef WITH_RPKI
+  if(rpki_input != NULL && rpki_input->rpki_active){
+    cfg_destroy(cfg);
+    bgpstream_rpki_destroy_input(rpki_input);
+  }
+#endif
+
  done:
   /* deallocate memory for interface */
   bgpstream_destroy(bs);
   return 0;
 
 err:
   bgpstream_destroy(bs);
+#ifdef WITH_RPKI
+  if(rpki_input != NULL && rpki_input->rpki_active){
+    cfg_destroy(cfg);
```

Comment:
I couldn't find the definition of `cfg_destroy`.

Reply:
```cfg_destroy``` is a function of the ROAFetchlib. I now added ```bgpstream_rpki_destroy_cfg``` to wrap the other one.